tools/mason/MasonHelp.chpl

```diff
@@ -159,3 +159,20 @@ proc masonTestHelp() {
   writeln("Tests pass if they exit with status code 0");
 }
 
+proc masonSystemHelp() {
+  writeln("Integrate a Mason package with system packages found via pkg-config");
+  writeln();
+  writeln("Usage:");
+  writeln('    mason system [options] [<args>...]');
+  writeln("    mason system [options]");
+  writeln();
+  writeln("Options:");
+  writeln("    pc                          Print a system packages .pc file");
+  writeln("    list                        List all packages available on the system");
```

Comment:
Should this actually be `search` to reflect `mason search`, where an empty `mason search` results in listing all available packages?